src/main/java/uk/gov/ons/ctp/response/casesvc/utility/PubSubEmulator.java

```diff
@@ -119,18 +120,4 @@ public void publishSampleCaseCreationMessage(String message) {
   public void shutdown() {
     CHANNEL.shutdown();
   }
-
-  public void testInit() {
-    Topic topic = topicClient.createTopic(topicName);
-    System.out.println("Created topic: " + topic.getName());
-    Subscription subscription =
-        subscriptionAdminClient.createSubscription(
-            subscriptionName, topicName, PushConfig.getDefaultInstance(), 10);
-    System.out.println("Created pull subscription: " + subscription.getName());
-  }
-
-  public void testTeardown() {
-    subscriptionAdminClient.deleteSubscription(subscriptionName);
-    topicClient.deleteTopic(topicName);
-  }
```

Comment:
For the non-integration test emulator, the Docker network version of RASRM has the topics and subscriptions created during the `docker-compose up` config.